index.html

```diff
@@ -332,17 +332,16 @@
           descriptor.
         </p>
         <p>
-          If the descriptor does not have a deviceId, its semantic is that
-          it queries for access to all devices of that class. Thus, if a query
-          for the "camera" permission with no deviceId returns "granted", the
+          If the descriptor does not have a deviceId, its semantic is that it
+          queries for access to all devices of that class. Thus, if a query for
+          the "camera" permission with no deviceId returns "granted", the
           client knows that there will never be a permission prompt for a
```

Comment:
This is really noisy. Why are we refactoring line length on areas where we're not making any changes?